tutorials/polkadot-sdk/build-a-blockchain/add-trusted-nodes.md

```diff
@@ -0,0 +1,313 @@
+
+    --8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/add-trusted-nodes/key-sr25519-1.html'
+
+    Note the Sr25519 public key for the account (SS58 format). This key will be used for producing blocks with `aura`. In this example, the Sr25519 public key for the account is `5HMhkSHpD4XcibjbU9ZiGemLpnsTUzLsG5JhQJQEcxp3KJaW`.
+
+3. Use the generated secret phrase to derive keys using the Ed25519 signature scheme. Enter the same password you used in the previous step:
+
+    ```bash
+    ./target/release/solochain-template-node key inspect \
+    --scheme Ed25519 \
+    --password-interactive \
+    "INSERT_SECRET_PHRASE"
+    ```
+
+    !!! note
+        Replace `INSERT_SECRET_PHRASE` with the secret phrase generated in step 2.
+
+    The command will output information about the generated keys similar to the following:
+
+    --8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/add-trusted-nodes/key-ed25519-1.html'
+
+    The Ed25519 key you've generated is crucial for block finalization using the `grandpa` consensus algorithm. The Ed25519 public key for the account is `5GdFMFbXy24uz8mFZroFUgdBkY2pq6igBNGAq9tsBfEZRSzP`.
+
+### Generate a Second Set of Keys
+
+In this tutorial, the private network will consist of two nodes, meaning you'll need two distinct sets of keys. You have several options for generating this second set of keys:
+
+- Use the keys from one of the predefined accounts
+- Follow the steps from the previous section, but use a different identity on your local machine to create a new key pair
+- Derive a child key pair to simulate a second identity on your local machine
+
+For this tutorial, the second set of keys will be:
+
+- Sr25519 (for Aura) - `5Df9bvnbqKNR8S1W2Uj5XSpJCKUomyymwCGf6WHKyoo3GDev`
+- Ed25519 (for Grandpa)  `5DJRQQWEaJart5yQnA6gnKLYKHLdpX6V4vHgzAYfNPT2NNuW`
+
+## Create a Custom Chain Specification
+
+After generating key pairs for your blockchain, the next step is creating a custom chain specification. You will share this specification with trusted validators participating in your network.
+
+To enable others to participate in your blockchain, ensure that each participant generates their own key pair. Once you collect the keys from all network participants, you can create a custom chain specification to replace the default local one.
+
+In this tutorial, you'll modify the local chain specification to create a custom version for a two-node network. The same process can be used to add more nodes if you have the necessary keys.
+
+### Steps to Create a Custom Chain Specification
+
+1. Open a terminal and navigate to the root directory of your compiled node template
+
+2. Export the local chain specification:
+
+    ```bash
+    ./target/release/solochain-template-node build-spec \
+    --disable-default-bootnode \
+    --chain local > customSpec.json
+    ```
+
+3. Preview the `customSpec.json` file:
+
+    - Preview first fields:
```

Comment:
Is there a command that should be run here? It should be in triple backticks so users can copy pasta